diff --git a/.gitignore b/.gitignore
index e6bbd7ae..7fe978f8 100644
--- a/.gitignore
+++ b/.gitignore
@@ -8,6 +8,7 @@
 .env
 .env.backup
 .env.production
+.phpunit.result.cache
 Homestead.json
 Homestead.yaml
 auth.json
diff --git a/CHANGELOG.md b/CHANGELOG.md
index a08dfc8a..abba1683 100644
--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,7 +1,16 @@
 # Release Notes
 
-## [Unreleased](https://github.com/laravel/laravel/compare/v9.5.1...10.x)
+## [Unreleased](https://github.com/laravel/laravel/compare/v10.0.2...10.x)
 
-## [v10.0.0 (2022-02-07)](https://github.com/laravel/laravel/compare/v9.5.1...10.x)
+## [v10.0.2](https://github.com/laravel/laravel/compare/v10.0.1...v10.0.2) - 2023-02-16
+
+- Remove unneeded call by @taylorotwell in https://github.com/laravel/laravel/commit/3986d4c54041fd27af36f96cf11bd79ce7b1ee4e
+
+## [v10.0.1](https://github.com/laravel/laravel/compare/v10.0.0...v10.0.1) - 2023-02-15
+
+- Add PHPUnit result cache to gitignore by @itxshakil in https://github.com/laravel/laravel/pull/6105
+- Allow php-http/discovery as a composer plugin by @nicolas-grekas in https://github.com/laravel/laravel/pull/6106
+
+## [v10.0.0 (2022-02-14)](https://github.com/laravel/laravel/compare/v9.5.2...v10.0.0)
 
 Laravel 10 includes a variety of changes to the application skeleton. Please consult the diff to see what's new.
diff --git a/app/Providers/AuthServiceProvider.php b/app/Providers/AuthServiceProvider.php
index 5522aa2f..dafcbee7 100644
--- a/app/Providers/AuthServiceProvider.php
+++ b/app/Providers/AuthServiceProvider.php
@@ -21,8 +21,6 @@ class AuthServiceProvider extends ServiceProvider
      */
     public function boot(): void
     {
-        $this->registerPolicies();
-
         //
     }
 }
diff --git a/composer.json b/composer.json
index 5b40f87c..4958668f 100644
--- a/composer.json
+++ b/composer.json
@@ -48,9 +48,6 @@
         ]
     },
     "extra": {
-        "branch-alias": {
-            "dev-master": "10.x-dev"
-        },
         "laravel": {
             "dont-discover": []
         }
@@ -60,7 +57,8 @@
         "preferred-install": "dist",
         "sort-packages": true,
         "allow-plugins": {
-            "pestphp/pest-plugin": true
+            "pestphp/pest-plugin": true,
+            "php-http/discovery": true
         }
     },
     "minimum-stability": "stable",
diff --git a/config/app.php b/config/app.php
index bca112fb..ef76a7ed 100644
--- a/config/app.php
+++ b/config/app.php
@@ -56,7 +56,7 @@ return [
 
     'url' => env('APP_URL', 'http://localhost'),
 
-    'asset_url' => env('ASSET_URL', '/'),
+    'asset_url' => env('ASSET_URL'),
 
     /*
     |--------------------------------------------------------------------------
diff --git a/config/mail.php b/config/mail.php
index 049052ff..542d98c3 100644
--- a/config/mail.php
+++ b/config/mail.php
@@ -28,7 +28,7 @@ return [
     | sending an e-mail. You will specify which one you are using for your
     | mailers below. You are free to add additional mailers as required.
     |
-    | Supported: "smtp", "sendmail", "mailgun", "ses",
+    | Supported: "smtp", "sendmail", "mailgun", "ses", "ses-v2",
     |            "postmark", "log", "array", "failover"
     |
     */
diff --git a/database/factories/UserFactory.php b/database/factories/UserFactory.php
index d4e88356..a6ecc0af 100644
--- a/database/factories/UserFactory.php
+++ b/database/factories/UserFactory.php
@@ -28,8 +28,6 @@ class UserFactory extends Factory
 
     /**
      * Indicate that the model's email address should be unverified.
-     *
-     * @return $this
      */
     public function unverified(): static
     {
