diff --git a/.gitignore b/.gitignore
index 46340a60..afa306bd 100644
--- a/.gitignore
+++ b/.gitignore
@@ -18,3 +18,4 @@ yarn-error.log
 /.fleet
 /.idea
 /.vscode
+/.zed
diff --git a/CHANGELOG.md b/CHANGELOG.md
index 2732f5b0..b5a318e5 100644
--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,6 +1,10 @@
 # Release Notes
 
-## [Unreleased](https://github.com/laravel/laravel/compare/v11.1.4...11.x)
+## [Unreleased](https://github.com/laravel/laravel/compare/v11.1.5...11.x)
+
+## [v11.1.5](https://github.com/laravel/laravel/compare/v11.1.4...v11.1.5) - 2024-08-14
+
+* Update axios by [@laserhybiz](https://github.com/laserhybiz) in https://github.com/laravel/laravel/pull/6440
 
 ## [v11.1.4](https://github.com/laravel/laravel/compare/v11.1.3...v11.1.4) - 2024-07-16
 
diff --git a/config/filesystems.php b/config/filesystems.php
index c5f244d7..b564035a 100644
--- a/config/filesystems.php
+++ b/config/filesystems.php
@@ -32,7 +32,8 @@ return [
 
         'local' => [
             'driver' => 'local',
-            'root' => storage_path('app'),
+            'root' => storage_path('app/private'),
+            'serve' => true,
             'throw' => false,
         ],
 
diff --git a/storage/app/.gitignore b/storage/app/.gitignore
index 8f4803c0..fedb287f 100644
--- a/storage/app/.gitignore
+++ b/storage/app/.gitignore
@@ -1,3 +1,4 @@
 *
+!private/
 !public/
 !.gitignore
diff --git a/storage/app/private/.gitignore b/storage/app/private/.gitignore
new file mode 100644
index 00000000..d6b7ef32
--- /dev/null
+++ b/storage/app/private/.gitignore
@@ -0,0 +1,2 @@
+*
+!.gitignore
