diff --git a/.editorconfig b/.editorconfig
index a186cd20..6df84280 100644
--- a/.editorconfig
+++ b/.editorconfig
@@ -14,5 +14,5 @@ trim_trailing_whitespace = false
 [*.{yml,yaml}]
 indent_size = 2
 
-[compose.yaml]
+[{compose,docker-compose}.{yml,yaml}]
 indent_size = 4
diff --git a/.gitignore b/.gitignore
index 0eeb578e..83d894f7 100644
--- a/.gitignore
+++ b/.gitignore
@@ -5,7 +5,8 @@
 .env.production
 .phpactor.json
 .phpunit.result.cache
-/.fleet
+/.codex
+/.cursor/
 /.idea
 /.nova
 /.phpunit.cache
diff --git a/.npmrc b/.npmrc
new file mode 100644
index 00000000..495a6af9
--- /dev/null
+++ b/.npmrc
@@ -0,0 +1,2 @@
+ignore-scripts=true
+audit=true
diff --git a/CHANGELOG.md b/CHANGELOG.md
index fbf513d9..531bde7b 100644
--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,6 +1,23 @@
 # Release Notes
 
-## [Unreleased](https://github.com/laravel/laravel/compare/v13.1.0...13.x)
+## [Unreleased](https://github.com/laravel/laravel/compare/v13.2.0...13.x)
+
+## [v13.2.0](https://github.com/laravel/laravel/compare/v13.1.2...v13.2.0) - 2026-04-09
+
+* Remove axios and enable ignore-scripts by [@WendellAdriel](https://github.com/WendellAdriel) in https://github.com/laravel/laravel/pull/6778
+* Add /.cursor/ to .gitignore by [@workwithbinu](https://github.com/workwithbinu) in https://github.com/laravel/laravel/pull/6782
+* Remove '.fleet' from .gitignore by [@dominiq007](https://github.com/dominiq007) in https://github.com/laravel/laravel/pull/6783
+* Support all compose file naming conventions in editorconfig by [@mmachatschek](https://github.com/mmachatschek) in https://github.com/laravel/laravel/pull/6786
+
+## [v13.1.2](https://github.com/laravel/laravel/compare/v13.1.1...v13.1.2) - 2026-03-31
+
+* Prevents installed package from executing malicious code via `postinstall` by [@crynobone](https://github.com/crynobone) in https://github.com/laravel/laravel/pull/6777
+* Add missing comma in axios by [@aziyan99](https://github.com/aziyan99) in https://github.com/laravel/laravel/pull/6779
+
+## [v13.1.1](https://github.com/laravel/laravel/compare/v13.1.0...v13.1.1) - 2026-03-31
+
+* Update .gitignore by [@Cegem-360](https://github.com/Cegem-360) in https://github.com/laravel/laravel/pull/6774
+* [security] pin axios version by [@NickSdot](https://github.com/NickSdot) in https://github.com/laravel/laravel/pull/6776
 
 ## [v13.1.0](https://github.com/laravel/laravel/compare/v12.12.2...v13.1.0) - 2026-03-18
 
diff --git a/composer.json b/composer.json
index 0422a1e2..41f874d3 100644
--- a/composer.json
+++ b/composer.json
@@ -36,7 +36,7 @@
             "@php -r \"file_exists('.env') || copy('.env.example', '.env');\"",
             "@php artisan key:generate",
             "@php artisan migrate --force",
-            "npm install",
+            "npm install --ignore-scripts",
             "npm run build"
         ],
         "dev": [
diff --git a/database/migrations/0001_01_01_000002_create_jobs_table.php b/database/migrations/0001_01_01_000002_create_jobs_table.php
index 425e7058..87d3ed78 100644
--- a/database/migrations/0001_01_01_000002_create_jobs_table.php
+++ b/database/migrations/0001_01_01_000002_create_jobs_table.php
@@ -15,7 +15,7 @@ return new class extends Migration
             $table->id();
             $table->string('queue')->index();
             $table->longText('payload');
-            $table->unsignedTinyInteger('attempts');
+            $table->unsignedSmallInteger('attempts');
             $table->unsignedInteger('reserved_at')->nullable();
             $table->unsignedInteger('available_at');
             $table->unsignedInteger('created_at');
diff --git a/package.json b/package.json
index 3aa557cf..296ac3f0 100644
--- a/package.json
+++ b/package.json
@@ -8,7 +8,6 @@
     },
     "devDependencies": {
         "@tailwindcss/vite": "^4.0.0",
-        "axios": ">=1.11.0 <=1.14.0"
         "concurrently": "^9.0.1",
         "laravel-vite-plugin": "^3.0.0",
         "tailwindcss": "^4.0.0",
diff --git a/resources/js/app.js b/resources/js/app.js
index e59d6a0a..8337712e 100644
--- a/resources/js/app.js
+++ b/resources/js/app.js
@@ -1 +1 @@
-import './bootstrap';
+//
diff --git a/resources/js/bootstrap.js b/resources/js/bootstrap.js
deleted file mode 100644
index 5f1390b0..00000000
--- a/resources/js/bootstrap.js
+++ /dev/null
@@ -1,4 +0,0 @@
-import axios from 'axios';
-window.axios = axios;
-
-window.axios.defaults.headers.common['X-Requested-With'] = 'XMLHttpRequest';
diff --git a/resources/views/welcome.blade.php b/resources/views/welcome.blade.php
index 5f83753b..2c2e7c27 100644
--- a/resources/views/welcome.blade.php
+++ b/resources/views/welcome.blade.php
@@ -120,7 +120,7 @@
 
                     <p class="mt-6 lg:mt-10 text-[#706f6c] dark:text-[#A1A09A]">
                         v{{ app()->version() }}
-                        <a href="https://github.com/laravel/laravel/blob/13.x/CHANGELOG.md" target="_blank" class="inline-flex items-center space-x-1 font-medium underline underline-offset-4 text-[#f53003] dark:text-[#FF4433] ml-1">
+                        <a href="https://github.com/laravel/framework/blob/13.x/CHANGELOG.md" target="_blank" class="inline-flex items-center space-x-1 font-medium underline underline-offset-4 text-[#f53003] dark:text-[#FF4433] ml-1">
                             <span>View changelog</span>
                             <svg
                                 width="10"
