diff --git a/CHANGELOG.md b/CHANGELOG.md
index fa89039e..80d51eb7 100644
--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,6 +1,15 @@
 # Release Notes
 
-## [Unreleased](https://github.com/laravel/laravel/compare/v8.4.2...8.x)
+## [Unreleased](https://github.com/laravel/laravel/compare/v8.4.3...8.x)
+
+
+## [v8.4.3 (2020-11-24)](https://github.com/laravel/laravel/compare/v8.4.2...v8.4.3)
+
+### Added
+- Add ably entry ([5182e9c](https://github.com/laravel/laravel/commit/5182e9c6de805e025fb4cfad63c210c3197002ab))
+
+### Fixed
+- Add missing null cache driver in `config/cache.php` ([#5472](https://github.com/laravel/laravel/pull/5472))
 
 
 ## [v8.4.2 (2020-11-17)](https://github.com/laravel/laravel/compare/v8.4.1...v8.4.2)
diff --git a/config/broadcasting.php b/config/broadcasting.php
index 3bba1103..ef208598 100644
--- a/config/broadcasting.php
+++ b/config/broadcasting.php
@@ -41,6 +41,11 @@ return [
             ],
         ],
 
+        'ably' => [
+            'driver' => 'ably',
+            'key' => env('ABLY_KEY'),
+        ],
+
         'redis' => [
             'driver' => 'redis',
             'connection' => 'default',
diff --git a/config/cache.php b/config/cache.php
index 7d5976ed..9473acc9 100644
--- a/config/cache.php
+++ b/config/cache.php
@@ -27,7 +27,7 @@ return [
     | same cache driver to group types of items stored in your caches.
     |
     | Supported drivers: "apc", "array", "database", "file",
-    |            "memcached", "redis", "dynamodb"
+    |            "memcached", "redis", "dynamodb", "null"
     |
     */
 
