diff --git a/.env.example b/.env.example
index 68c9578c..c3ed2a91 100644
--- a/.env.example
+++ b/.env.example
@@ -8,8 +8,7 @@ LOG_CHANNEL=stack
 LOG_LEVEL=debug
 
 DB_CONNECTION=mysql
-# DB_HOST=127.0.0.1
-DB_HOST=mysql
+DB_HOST=127.0.0.1
 DB_PORT=3306
 DB_DATABASE=laravel
 DB_USERNAME=root
@@ -21,11 +20,9 @@ QUEUE_CONNECTION=sync
 SESSION_DRIVER=file
 SESSION_LIFETIME=120
 
-# MEMCACHED_HOST=127.0.0.1
-MEMCACHED_HOST=memcached
+MEMCACHED_HOST=127.0.0.1
 
-# REDIS_HOST=127.0.0.1
-REDIS_HOST=redis
+REDIS_HOST=127.0.0.1
 REDIS_PASSWORD=null
 REDIS_PORT=6379
 
diff --git a/.gitignore b/.gitignore
index 0f7df0fb..0ae59f0b 100644
--- a/.gitignore
+++ b/.gitignore
@@ -6,6 +6,7 @@
 .env
 .env.backup
 .phpunit.result.cache
+docker-compose.override.yml
 Homestead.json
 Homestead.yaml
 npm-debug.log
diff --git a/README.md b/README.md
index 2f7ddcc6..e1dcd2e1 100644
--- a/README.md
+++ b/README.md
@@ -42,6 +42,7 @@ We would like to extend our thanks to the following sponsors for funding Laravel
 - **[Many](https://www.many.co.uk)**
 - **[Webdock, Fast VPS Hosting](https://www.webdock.io/en)**
 - **[DevSquad](https://devsquad.com)**
+- **[Curotec](https://www.curotec.com/)**
 - **[OP.GG](https://op.gg)**
 
 ## Contributing
diff --git a/app/Http/Kernel.php b/app/Http/Kernel.php
index 21a8754b..30020a50 100644
--- a/app/Http/Kernel.php
+++ b/app/Http/Kernel.php
@@ -14,7 +14,7 @@ class Kernel extends HttpKernel
      * @var array
      */
     protected $middleware = [
-        \App\Http\Middleware\TrustHosts::class,
+        // \App\Http\Middleware\TrustHosts::class,
         \App\Http\Middleware\TrustProxies::class,
         \Fruitcake\Cors\HandleCors::class,
         \App\Http\Middleware\PreventRequestsDuringMaintenance::class,
diff --git a/docker-compose.yml b/docker-compose.yml
index 089e4856..5ba8e628 100644
--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -32,7 +32,7 @@ services:
     mysql:
         image: 'mysql:8.0'
         ports:
-            - '${DB_PORT}:3306'
+            - '${FORWARD_DB_PORT:-3306}:3306'
         environment:
             MYSQL_ROOT_PASSWORD: '${DB_PASSWORD}'
             MYSQL_DATABASE: '${DB_DATABASE}'
@@ -46,7 +46,7 @@ services:
     redis:
         image: 'redis:alpine'
         ports:
-            - '${REDIS_PORT}:6379'
+            - '${FORWARD_REDIS_PORT:-6379}:6379'
         volumes:
             - 'sailredis:/data'
         networks:
