diff --git a/CHANGELOG.md b/CHANGELOG.md
index d83acbe0..c3efa307 100644
--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,6 +1,12 @@
 # Release Notes
 
-## [Unreleased](https://github.com/laravel/laravel/compare/v8.5.19...8.x)
+## [Unreleased](https://github.com/laravel/laravel/compare/v8.5.20...8.x)
+
+
+## [v8.5.20 (2021-06-15)](https://github.com/laravel/laravel/compare/v8.5.19...v8.5.20)
+
+### Changed
+- Add translation for current_password rule ([#5628](https://github.com/laravel/laravel/pull/5628))
 
 
 ## [v8.5.19 (2021-06-01)](https://github.com/laravel/laravel/compare/v8.5.18...v8.5.19)
diff --git a/resources/lang/en/validation.php b/resources/lang/en/validation.php
index 49e3388b..7a15f437 100644
--- a/resources/lang/en/validation.php
+++ b/resources/lang/en/validation.php
@@ -31,6 +31,7 @@ return [
     ],
     'boolean' => 'The :attribute field must be true or false.',
     'confirmed' => 'The :attribute confirmation does not match.',
+    'current_password' => 'The password is incorrect.',
     'date' => 'The :attribute is not a valid date.',
     'date_equals' => 'The :attribute must be a date equal to :date.',
     'date_format' => 'The :attribute does not match the format :format.',
