diff --git a/.gitattributes b/.gitattributes
index 967315dd..510d9961 100644
--- a/.gitattributes
+++ b/.gitattributes
@@ -1,5 +1,10 @@
 * text=auto
-*.css linguist-vendored
-*.scss linguist-vendored
-*.js linguist-vendored
+
+*.blade.php diff=html
+*.css diff=css
+*.html diff=html
+*.md diff=markdown
+*.php diff=php
+
+/.github export-ignore
 CHANGELOG.md export-ignore
diff --git a/.github/workflows/update-changelog.yml b/.github/workflows/update-changelog.yml
new file mode 100644
index 00000000..0200e2b9
--- /dev/null
+++ b/.github/workflows/update-changelog.yml
@@ -0,0 +1,29 @@
+name: "Update Changelog"
+
+on:
+  release:
+    types: [released]
+
+jobs:
+  update:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.ref_name }}
+
+      - name: Update Changelog
+        uses: stefanzweifel/changelog-updater-action@v1
+        with:
+          latest-version: ${{ github.event.release.tag_name }}
+          release-notes: ${{ github.event.release.body }}
+          compare-url-target-revision: ${{ github.event.release.target_commitish }}
+
+      - name: Commit updated CHANGELOG
+        uses: stefanzweifel/git-auto-commit-action@v4
+        with:
+          branch: ${{ github.event.release.target_commitish }}
+          commit_message: Update CHANGELOG.md
+          file_pattern: CHANGELOG.md
diff --git a/CHANGELOG.md b/CHANGELOG.md
index eabd7a59..0c859aa2 100644
--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,6 +1,18 @@
 # Release Notes
 
-## [Unreleased](https://github.com/laravel/laravel/compare/v8.6.9...8.x)
+## [Unreleased](https://github.com/laravel/laravel/compare/v8.6.10...8.x)
+
+
+## [v8.6.10 (2021-12-22)](https://github.com/laravel/laravel/compare/v8.6.9...v8.6.10)
+
+### Changed
+- Bump Laravel to v8.75 ([#5750](https://github.com/laravel/laravel/pull/5750))
+- Simplify the maintenance file call ([#5752](https://github.com/laravel/laravel/pull/5752))
+- Add enum translation ([#5753](https://github.com/laravel/laravel/pull/5753))
+- Add mac_address validation message ([#5754](https://github.com/laravel/laravel/pull/5754))
+
+### Removed
+- Delete web.config ([#5744](https://github.com/laravel/laravel/pull/5744))
 
 
 ## [v8.6.9 (2021-12-07)](https://github.com/laravel/laravel/compare/v8.6.8...v8.6.9)
diff --git a/README.md b/README.md
index 8878ec11..1b6397ce 100644
--- a/README.md
+++ b/README.md
@@ -44,10 +44,8 @@ We would like to extend our thanks to the following sponsors for funding Laravel
 - **[DevSquad](https://devsquad.com)**
 - **[Curotec](https://www.curotec.com/services/technologies/laravel/)**
 - **[OP.GG](https://op.gg)**
-- **[CMS Max](https://www.cmsmax.com/)**
 - **[WebReinvent](https://webreinvent.com/?utm_source=laravel&utm_medium=github&utm_campaign=patreon-sponsors)**
 - **[Lendio](https://lendio.com)**
-- **[Romega Software](https://romegasoftware.com)**
 
 ## Contributing
 
diff --git a/composer.json b/composer.json
index 8fc7406e..2b0c1155 100644
--- a/composer.json
+++ b/composer.json
@@ -8,7 +8,7 @@
         "php": "^7.3|^8.0",
         "fruitcake/laravel-cors": "^2.0",
         "guzzlehttp/guzzle": "^7.0.1",
-        "laravel/framework": "^8.65",
+        "laravel/framework": "^8.75",
         "laravel/sanctum": "^2.11",
         "laravel/tinker": "^2.5"
     },
diff --git a/config/auth.php b/config/auth.php
index e29a3f7d..d8c6cee7 100644
--- a/config/auth.php
+++ b/config/auth.php
@@ -80,7 +80,7 @@ return [
     | than one user table or model in the application and you want to have
     | separate password reset settings based on the specific user types.
     |
-    | The expire time is the number of minutes that the reset token should be
+    | The expire time is the number of minutes that each reset token will be
     | considered valid. This security feature keeps tokens short-lived so
     | they have less time to be guessed. You may change this as needed.
     |
diff --git a/config/hashing.php b/config/hashing.php
index 84257708..bcd3be4c 100644
--- a/config/hashing.php
+++ b/config/hashing.php
@@ -44,9 +44,9 @@ return [
     */
 
     'argon' => [
-        'memory' => 1024,
-        'threads' => 2,
-        'time' => 2,
+        'memory' => 65536,
+        'threads' => 1,
+        'time' => 4,
     ],
 
 ];
diff --git a/public/index.php b/public/index.php
index 002ee24d..1d69f3a2 100644
--- a/public/index.php
+++ b/public/index.php
@@ -16,8 +16,8 @@ define('LARAVEL_START', microtime(true));
 |
 */
 
-if (file_exists(__DIR__.'/../storage/framework/maintenance.php')) {
-    require __DIR__.'/../storage/framework/maintenance.php';
+if (file_exists($maintenance = __DIR__.'/../storage/framework/maintenance.php')) {
+    require $maintenance;
 }
 
 /*
diff --git a/public/web.config b/public/web.config
deleted file mode 100644
index 323482f1..00000000
--- a/public/web.config
+++ /dev/null
@@ -1,28 +0,0 @@
-<!--
-    Rewrites requires Microsoft URL Rewrite Module for IIS
-    Download: https://www.iis.net/downloads/microsoft/url-rewrite
-    Debug Help: https://docs.microsoft.com/en-us/iis/extensions/url-rewrite-module/using-failed-request-tracing-to-trace-rewrite-rules
--->
-<configuration>
-  <system.webServer>
-    <rewrite>
-      <rules>
-        <rule name="Imported Rule 1" stopProcessing="true">
-          <match url="^(.*)/$" ignoreCase="false" />
-          <conditions>
-            <add input="{REQUEST_FILENAME}" matchType="IsDirectory" ignoreCase="false" negate="true" />
-          </conditions>
-          <action type="Redirect" redirectType="Permanent" url="/{R:1}" />
-        </rule>
-        <rule name="Imported Rule 2" stopProcessing="true">
-          <match url="^" ignoreCase="false" />
-          <conditions>
-            <add input="{REQUEST_FILENAME}" matchType="IsDirectory" ignoreCase="false" negate="true" />
-            <add input="{REQUEST_FILENAME}" matchType="IsFile" ignoreCase="false" negate="true" />
-          </conditions>
-          <action type="Rewrite" url="index.php" />
-        </rule>
-      </rules>
-    </rewrite>
-  </system.webServer>
-</configuration>
diff --git a/resources/lang/en/validation.php b/resources/lang/en/validation.php
index ba42c8d9..783003cf 100644
--- a/resources/lang/en/validation.php
+++ b/resources/lang/en/validation.php
@@ -45,6 +45,7 @@ return [
     'distinct' => 'The :attribute field has a duplicate value.',
     'email' => 'The :attribute must be a valid email address.',
     'ends_with' => 'The :attribute must end with one of the following: :values.',
+    'enum' => 'The selected :attribute is invalid.',
     'exists' => 'The selected :attribute is invalid.',
     'file' => 'The :attribute must be a file.',
     'filled' => 'The :attribute field must have a value.',
@@ -80,6 +81,7 @@ return [
         'string' => 'The :attribute must be less than or equal to :value characters.',
         'array' => 'The :attribute must not have more than :value items.',
     ],
+    'mac_address' => 'The :attribute must be a valid MAC address.',
     'max' => [
         'numeric' => 'The :attribute must not be greater than :max.',
         'file' => 'The :attribute must not be greater than :max kilobytes.',
@@ -106,6 +108,7 @@ return [
     'prohibits' => 'The :attribute field prohibits :other from being present.',
     'regex' => 'The :attribute format is invalid.',
     'required' => 'The :attribute field is required.',
+    'required_array_keys' => 'The :attribute field must contain entries for: :values.',
     'required_if' => 'The :attribute field is required when :other is :value.',
     'required_unless' => 'The :attribute field is required unless :other is in :values.',
     'required_with' => 'The :attribute field is required when :values is present.',
