diff --git a/CHANGELOG.md b/CHANGELOG.md
index e39743c5..0e15e5de 100644
--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,6 +1,13 @@
 # Release Notes
 
-## [Unreleased](https://github.com/laravel/laravel/compare/v9.1.4...9.x)
+## [Unreleased](https://github.com/laravel/laravel/compare/v9.1.5...9.x)
+
+## [v9.1.5](https://github.com/laravel/laravel/compare/v9.1.4...v9.1.5) - 2022-04-12
+
+### Changed
+
+- Rearrange route methods by @osbre in https://github.com/laravel/laravel/pull/5862
+- Add levels to handler by @taylorotwell in https://github.com/laravel/laravel/commit/a507e1424339633ce423729ec0ac49b99f0e57d7
 
 ## [v9.1.4](https://github.com/laravel/laravel/compare/v9.1.3...v9.1.4) - 2022-03-29
 
diff --git a/config/app.php b/config/app.php
index b02c7f4b..ef76a7ed 100644
--- a/config/app.php
+++ b/config/app.php
@@ -125,6 +125,24 @@ return [
 
     'cipher' => 'AES-256-CBC',
 
+    /*
+    |--------------------------------------------------------------------------
+    | Maintenance Mode Driver
+    |--------------------------------------------------------------------------
+    |
+    | These configuration options determine the driver used to determine and
+    | manage Laravel's "maintenance mode" status. The "cache" driver will
+    | allow maintenance mode to be controlled across multiple machines.
+    |
+    | Supported drivers: "file", "cache"
+    |
+    */
+
+    'maintenance' => [
+        'driver' => 'file',
+        // 'store'  => 'redis',
+    ],
+
     /*
     |--------------------------------------------------------------------------
     | Autoloaded Service Providers
diff --git a/lang/en.json b/lang/en.json
deleted file mode 100644
index 577680dd..00000000
--- a/lang/en.json
+++ /dev/null
@@ -1,7 +0,0 @@
-{
-    "The :attribute must contain at least one letter.": "The :attribute must contain at least one letter.",
-    "The :attribute must contain at least one number.": "The :attribute must contain at least one number.",
-    "The :attribute must contain at least one symbol.": "The :attribute must contain at least one symbol.",
-    "The :attribute must contain at least one uppercase and one lowercase letter.": "The :attribute must contain at least one uppercase and one lowercase letter.",
-    "The given :attribute has appeared in a data leak. Please choose a different :attribute.": "The given :attribute has appeared in a data leak. Please choose a different :attribute."
-}
diff --git a/lang/en/validation.php b/lang/en/validation.php
index 397f78e2..8ca5a008 100644
--- a/lang/en/validation.php
+++ b/lang/en/validation.php
@@ -100,6 +100,13 @@ return [
     'not_in' => 'The selected :attribute is invalid.',
     'not_regex' => 'The :attribute format is invalid.',
     'numeric' => 'The :attribute must be a number.',
+    'password' => [
+        'mixed' => 'The :attribute must contain at least one uppercase and one lowercase letter.',
+        'letters' => 'The :attribute must contain at least one letter.',
+        'symbols' => 'The :attribute must contain at least one symbol.',
+        'numbers' => 'The :attribute must contain at least one number.',
+        'uncompromised' => 'The given :attribute has appeared in a data leak. Please choose a different :attribute.',
+    ],
     'present' => 'The :attribute field must be present.',
     'prohibited' => 'The :attribute field is prohibited.',
     'prohibited_if' => 'The :attribute field is prohibited when :other is :value.',
